Fitstrailer/Story/Program/ProgramView.swift

```diff
@@ -21,9 +21,10 @@ struct ProgramView: View {
                 }
                 .padding()
                 List(program.days) { day in
-                    NavigationLink(destination: DayRowView(day: day)) {
+                    NavigationLink(destination: DailyProgramView()) {
                         DayRowView(day: day)
```

Comment:
Burada DayRowView, spesifik bir day için olan bilgileri gösteriyor. 
Ancak destination ile gidilen view'da hangi günlük program olduğu belirtilmemiş.
Bu day in DailyProgramView'da olması ve DailyProgramView'da sadece ilgili günün bilgilerinin gösterilmesi gerekir.
Böyle, nereden gidersek gidelim aynı view i göreceğiz.